README.md

```diff
@@ -23,145 +23,165 @@ which functions as the data backbone for prominent hedge funds, banks, and finan
 
 | Language      | Server Application | Client Application (OpenAPI) |
 | ------------- | ------------------ | ---------------------------- |
-| Python        | Yes                | No                           |
-| Java / Groovy | Yes                | No                           |
+| Python        | Yes                | Yes                           |
+| Java / Groovy | Yes                | Yes                           |
 | JavaScript    | No                 | Yes                          |
 | gRPC          | -                  | Yes                          |
 
-## Running Deephaven
+## Run Deephaven
+
+This section is a quick start guide for running Deephaven from pre-built images.
+For detailed instructions on running Deephaven, see [Launch Deephaven from pre-built images](https://deephaven.io/core/docs/tutorials/launch-pre-built).
```

Comment:
```suggestion
For detailed instructions on running Deephaven, see [Launch Deephaven from pre-built images](https://deephaven.io/core/docs/tutorials/launch-pre-built). The pre-built images are for normal Deephaven deployments.
```

Reply:
Are we sure most users will know what "normal Deephaven deployments" are?  This feels awkward when coupled with the line below that says "if you're not sure, use this one."  Does this work better?

```suggestion
Standard users should build and run Deephaven from pre-built images.  For detailed instructions, see [Launch Deephaven from pre-built images](https://deephaven.io/core/docs/tutorials/launch-pre-built).
```

Reply:
I think "standard users" has a similar problem.  Ugh.